applications/GeoMechanicsApplication/tests/cpp_tests/custom_constitutive/test_linear_elastic_3D_law.cpp

```diff
@@ -0,0 +1,175 @@
+// KRATOS___
```

Comment:
The incremental nature of the law is not tested yet. That would require two steps and change of the Young's modulus in between, to distinguish between an incremental and a total formulation.

Reply:
It is tested for the 2D law though, since the incremental nature is not really dimension-specific, that could be enough, let's have a discussion on this.

Reply:
I should have read more carefully, Richard has shown me that the testing is there also for 3D.